src/signalSTM.h

```diff
@@ -0,0 +1,354 @@
+#pragma once
+
+#if !defined(__AVR__) && defined(ARDUINO_MAPLEMINI_F103CB) || defined(STM32F103xB)
+#include "compile_config.h"
+
+/*
+
+*   developer:
+*   2020 S.Butzek, HomeAutoUser, elektron-bbs
+
+*   Integration for compatibility with a similar project by Ralf9
+
+*   tested hardware:
+*    - Maple Mini STM32F103CBT6
+
+*/
+
+#define PROGNAME               " SIGNALduino_STM32 "
+#define PROGVERS               "3.4.0-dev_20200626"
```

Comment:
Da müsste uns mal was einfallen, dass wir die Version nicht mehrfach pflegen. Aber das war schon beim ESP ein Thema erinnere ich mich

Reply:
Ich bin hier schon dran gewesen und habe geteset aber bisher führte noch nichts zum Erfolg.

Das einzige was mir einfallen würde, man verschiebt die Versionsnummer in die 
`compile_config.h` und hätte nur eine Stelle zum pflegen.